ocis/pkg/runtime/service/service.go

```diff
@@ -248,12 +248,24 @@ func (s *Service) generateRunSet(cfg *ociscfg.Config) {
 		return
 	}
 
+	disabled := make(map[string]bool)
+	if cfg.Runtime.Disabled != "" {
+		e := strings.Split(strings.ReplaceAll(cfg.Runtime.Disabled, " ", ""), ",")
+		for _, s := range e {
+			disabled[s] = true
```

Comment:
Why do the extra step and store it to a separate array, can't you just remove it from the `runset` above?

Reply:
`runset` is a `[]string`. To delete from it I would need to loop over it to find the index, then delete the index from the slice. There are some other tricks how to delete from slices but all of them feel hacky to me. I think it is much cleaner with a helping map.

Reply:
Yes that works. Good idea. :+1: Now the name also makes sense because it is really a `set`